Gems/ROS2/Code/ros2_files.cmake

```diff
@@ -1,14 +1,14 @@
 
 set(FILES
     Include/ROS2/ROS2Bus.h
-    Source/LidarRaycaster.cpp
-    Source/LidarRaycaster.h
-    Source/LidarTemplate.h
-    Source/ROS2LidarSensorComponent.cpp
-    Source/ROS2LidarSensorComponent.h
+        Source/Lidar/LidarRaycaster.cpp
```

Comment:
Is this formatting correct?
Looks like the files paths are not aligned.

Reply:
My Clion needs to be tamed (an effect of auto refactor when dropping in folder). Fixed